release-1.8/release-1.8.md

```diff
@@ -0,0 +1,79 @@
+# Proposed timeline for 1.8
+We will begin the 1.8 release the first week of July 2017.
+
+The proposal below is similar in layout to the 1.7 plans with some
+modifications:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+
+## 1.8 Release Schedule
+- July 5 (Weds) coding start (7w)
+- July 12th (Weds): v1.8.0-alpha.1
+- July 26th (Weds): v1.8.0-alpha.2
+- Aug 1 (Tues): Feature Freeze
+- Aug 9 (Weds): v1.8.0-alpha.3
+- Aug 23 (Weds): v1.8.0-alpha.4
+- Sept 1 (Fri): Code Freeze 
+- Sept 6 (Weds): v1.8.0-beta.0, v1.9.0-alpha.0
+- Sept 13 (Weds): v1.8.0-beta.1
+- Sept 18th (Mon): v1.9.0-alpha.1
+- Sept 20th (Weds): v1.8.release-candidate
+- Sept 27th (Weds):  v1.8.0 (release!)
+
+## 1.8 Details
+
+### July 5 - Sept 1
+- 8 week coding period
+- Release 1.8 alphas every 2 weeks
+- July 12th (Weds): v1.8.0-alpha.1
+- July 26th (Weds): v1.8.0-alpha.2
+- Aug 1st (Tues): Feature Freeze: all features going into release should
+  have an associated issue in the features repo
+- Aug 9th (Weds): v1.8.0-alpha.3
+- Aug 23rd (Weds): v1.8.0-alpha.4
+
+### Sept 1 (Fri) - Sept 14 (Thurs)
+- Sept 1st Begin Code Freeze
+  * Hard deadline for feature-related PRs to be in submit queue
+  * Add milestone restriction on submit queue.
+  * Community Feature Burndown Meetings held two or three times until release week (then every day). For those interested in joining please join [Kubernetes Milestone Burndown Group](https://groups.google.com/forum/#!forum/kubernetes-milestone-burndown)
+  * Focus on bugfix, test flakes and stabilization
+  * Ensure docs and release notes are written
+  * Identify all features going into the release, and make sure alpha, beta, ga is marked in features repo
+- Sept 6th (Weds): v1.8.0-beta.0
+  * Branch Manager to create v1.8 release branch.
+  * v1.9.0-alpha.0 is cut
+  * Test-infra lead to set up CI for branch.
+  * Begin regular fast-forwards (at least daily to keep CI up-to-date).
+- Sept 13th (Weds): End of Code Slush
+  * v1.8.0-beta.1 cut
+  * Final fast-forward of release branch.
+  * After this, all changes for the release must now be cherrypicked in batches by branch
+  manager.
+  * Remove milestone restriction on submit queue. Bot drains backlog over the
+  weekend.
+
+### Sept 15 - Sept 27
+- Sept 18th (Mon): v1.9.0-alpha.1
+- Sept 20th (Weds): v1.8.release-candidate
+  * RC means no known release-blockers outstanding.
+  * Only accept cherrypicks for release-blockers.
+  * Announce on mailing lists, Twitter, etc. to ask people to build and test the RC and submit feedback
+  * Perhaps managed k8s providers make rc.1 available on early access channels.
+  * More RCs as needed to verify release-blocking fixes.
```

Comment:
Honestly, at 1 week out, half the 22 docs were merged and good to go. A couple days later (that Fri), we were at 80%. Even the day of the original release date, we were scrambling to get a couple docs reviewed and merged because one snuck in last minute and another one hadn't been completed and ready for review until that day. It follows sort of an exponential decay pattern. We can set the deadlines earlier, but there will still be a few stragglers.